src/core/ext/xds/xds_http_rbac_filter.cc

```diff
@@ -412,6 +438,32 @@ Json ParseHttpRbacToJson(const envoy_extensions_filters_http_rbac_v3_RBAC* rbac,
       }
       inner_rbac_json.emplace("policies", std::move(policies_object));
     }
+    // Flatten the nested messages defined in rbac.proto
+    if (envoy_config_rbac_v3_RBAC_has_audit_logging_options(rules)) {
```

Comment:
I just realized that we need env var protection for this code that looks at the new xDS fields.  Can you please put together a PR to add that ASAP?  I want to make sure that this PR doesn't make its way into a release branch before we have the env var protection, or else we'll have to deal with broken releases in the wild, which is a major pain.

Reply:
Ack. Will work on it right now.